editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/l10n/messages.properties

```diff
@@ -46,9 +46,11 @@ newProjectWizardRuntimePageRuntimeNewButtonDescription = Use this button to crea
 newProjectWizardRuntimePageCamelGroupLabel = Camel Version
 newProjectWizardRuntimePageCamelLabel = Select the Camel version for your new project
 newProjectWizardRuntimePageCamelDescription = Select a Camel version from our list of supported versions or enter a different one.
+newProjectWizardRuntimePageCamelVersionValidationLabel = Verify
+newProjectWizardRuntimePageCamelVersionValidationDescription = Click this button to verify you can access the version of Camel you selected in the version field.
 newProjectWizardRuntimePageNoRuntimeSelectedLabel = No Runtime selected
 newProjectWizardRuntimePageCamelVersionsDontMatchWarning = \nThe selected target runtime contains a different version of Apache Camel. You should align with the version installed in the runtime ({0}) or select a different runtime which supports your selected Camel version. You may continue at your own risk.
-newProjectWizardRuntimePageCamelVersionInvalidWarning = \nThe selected Apache Camel version {0} seems to be unavailable. Possible reasons are a missing internet connection or the Camel version simply does not exist. Please choose a different version.
+newProjectWizardRuntimePageCamelVersionInvalidWarning = \nThe selected Apache Camel version {0} seems to be unavailable. Possible reasons are a missing internet connection or the Camel version does not exist. Please choose a different version.
```

Comment:
another reason is that user needs to configure a specific maven repository @, isn't it?